Dockerfile

```diff
@@ -32,7 +30,35 @@ RUN ln -sfT /dev/stderr /var/log/nginx/error.log; \
 	ln -sfT /dev/stdout /var/log/nginx/access.log; \
 	chown -R --no-dereference www-data:adm /var/log/nginx/
 
-COPY --from=curlimpersonate /usr/local/lib/libcurl-impersonate-ff.so /usr/local/lib/curl-impersonate/
+# install curl-impersonate library
+RUN set -xe \
+    && apt-get update \
+    && apt-get install --assume-yes curl \
+    && curlimpersonate_version=0.6.0 \
+    && { \
+        { \
+            [ $(arch) = 'aarch64' ] \
+            && archive="libcurl-impersonate-v${curlimpersonate_version}.aarch64-linux-gnu.tar.gz" \
+            && sha512sum="d04b1eabe71f3af06aa1ce99b39a49c5e1d33b636acedcd9fad163bc58156af5c3eb3f75aa706f335515791f7b9c7a6c40ffdfa47430796483ecef929abd905d" \
+        ; } \
+        || { \
+            [ $(arch) = 'armv7l' ] \
+            && archive="libcurl-impersonate-v${curlimpersonate_version}.arm-linux-gnueabihf.tar.gz" \
+            && sha512sum="05906b4efa1a6ed8f3b716fd83d476b6eea6bfc68e3dbc5212d65a2962dcaa7bd1f938c9096a7535252b11d1d08fb93adccc633585ff8cb8cec5e58bfe969bc9" \
+        ; } \
+        || { \
+            [ $(arch) = 'x86_64' ] \
+            && archive="libcurl-impersonate-v${curlimpersonate_version}.x86_64-linux-gnu.tar.gz" \
+            && sha512sum="480bbe9452cd9aff2c0daaaf91f1057b3a96385f79011628a9237223757a9b0d090c59cb5982dc54ea0d07191657299ea91ca170a25ced3d7d410fcdff130ace" \
+        ; } \
+    } \
+    && curl -LO "https://github.com/lwthiker/curl-impersonate/releases/download/v${curlimpersonate_version}/${archive}" \
+    && echo "$sha512sum  $archive" | sha512sum -c - \
+    && mkdir -p /usr/local/lib/curl-impersonate \
+    && tar xaf "$archive" -C /usr/local/lib/curl-impersonate --wildcards 'libcurl-impersonate-ff.so*' \
+    && rm "$archive" \
+    && apt-get purge --assume-yes curl
```

Comment:
```suggestion
    && apt-get purge --assume-yes curl \
    && rm -rf /var/lib/apt/lists/*
```

Clean Apt cache

Reply:
Actually, we already have an apt update and clean higher up. It could be nice to optimise by doing it only once

Reply:
Maybe consider moving all this section up to line 25 to avoid having to do an apt update again.

Reply:
Yes, makes sense. I'll update the PR.